scipy-notebook/Dockerfile

```diff
@@ -37,7 +37,10 @@ RUN conda install --quiet --yes \
     'bokeh=0.11*' \
     'sqlalchemy=1.0*' \
     'hdf5=1.8.17' \
-    'h5py=2.6*' && \
+    'h5py=2.6*' \
+    'vincent=0.4.4*' \
+    'beautifulsoup4=4.5.1*' \
```

Comment:
One more small request that I missed before: would you mind removing the patch part of the version number so that it's allowed to vary like in the other package specs:

```
'vincent=0.4.*' \
'beautifulsoup4=4.5.*' \
```

Same with the packages in the Python2 environment below.